packages/shared/src/hooks/useOrganizationList.tsx

```diff
@@ -43,3 +195,12 @@ function createOrganizationList(organizationMemberships: OrganizationMembershipR
     organization: organizationMembership.organization,
   }));
 }
+
+function cacheKey(type: 'userInvitations', user: UserResource, pagination: GetUserOrganizationInvitations) {
```

Comment:
Do we care about typing these objects here? Do you think this would be enough? If it's generic enough in can be used in other similar places as well - other hooks using `swr` for example.
```suggestion
function cacheKey(...keys: Array<string | number | undefined>) {
```

Reply:
I think you are correct. Although swr@2 seems to automatically parse objects as keys, which will make this function obsolete

Reply:
We need to make sure to check whether `swr` uses referential equality checks or if it somehow builds a scalar value by paring the object's values - otherwise we might risk invalidating the cache on every render.

Reply:
Does [this](https://swr.vercel.app/docs/arguments#passing-objects) answer your question ?